static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -76,45 +84,83 @@
 
     @include mq($until: mobileLandscape) {
         font-size: 5vw;
-        padding-left: ($gs-gutter / 2);
-        padding-right: ($gs-gutter / 2);
+        line-height: 1.6;
         // Position after pseudo element which forces line break
         order: 1;
         justify-content: space-between;
     }
+
+    @include mq(mobileLandscape) {
+        font-size: 24px;
+    }
 }
 
+// Specific values for menu button breakpoints
+$veggieburgerSmall: 42px;
```

Comment:
I believe most of our scss variables are hyphenated instead of camel case so `$veggie-burger-small`